docs/RowOriented.md

```diff
@@ -68,6 +68,117 @@ using (var rowReader = ParquetFile.CreateRowReader<MyRow>("example.parquet"))
 }
 ```
 
+## Reading and writing custom types
+
+The row-oriented API supports reading and writing custom types by providing a `LogicalReadConverterFactory` or `LogicalWriteConverterFactory`.
```

Comment:
We should still mention the `LogicalTypeFactory` here:
```suggestion
The row-oriented API supports reading and writing custom types by providing a `LogicalTypeFactory` and a `LogicalReadConverterFactory` or `LogicalWriteConverterFactory`.
```

Reply:
That makes more sense :)